optimizely/optimizely_config.py

```diff
@@ -136,9 +117,127 @@ def __init__(self, project_config):
         self.environment_key = project_config.environment_key
         self.attributes = project_config.attributes
         self.events = project_config.events
+        self.rollouts = project_config.rollouts
 
         self._create_lookup_maps()
 
+        '''
+            Merging typed_audiences with audiences from project_config.
+            The typed_audiences has higher precedence.
+        '''
+
+        typed_audiences = project_config.typed_audiences[:]
```

Comment:
@The-inside-man Minor, but we don't mutate typed_audiences and thus can do a reference copy.

Reply:
Good point! Thanks for catching that!